mdata/chunk/format.go

```diff
@@ -10,4 +10,5 @@ type Format uint8
 // identifier of message format
 const (
 	FormatStandardGoTsz Format = iota
+	FormatWithLen
```

Comment:
since this is an extension of FormatStandardGoTsz, that also uses the same go-tsz encoding, but with the extra len field, this would be better named something like `FormatStandardGoTszWithLen`